### DISCUSSION TITLE
Kedro is fantastic. better live logging features would push it closer to an all in one

### DISCUSSION THREAD

**Discussion by eddyojb88** [kedro-org/kedro#3617] [Idea]
Hi.

Thank you for such a great package. It is the most professional I have come across having tested Metaflow, Kubeflow, GoKart (Luigi), MLFlow and Airflow.

However, one feature that I think it really needs is better logging in the UI for long running processes, similar to other async tools like Airflow, Celery Flower, Luigi etc. Hopefully I haven't made a mistake and missed this but is this on the roadmap?

Ed

**Comment by astrojuanlu** [comment]
Hi @eddyojb88 ! This is not in the roadmap for now. If anything, that would go into Kedro Viz, but we're trying to make Kedro play well with existing orchestrators and platforms instead.

**Comment by eddy-oj** [comment]
I have solved the issue of task monitoring, model serving, scalability and included an equivalent for Kedro Viz, task all with 1% of the code base. The package is called Django Flow Forge. Much simpler and more flexible,

**Comment by eddyojb88** [comment]
https://github.com/eddyojb88/django-flow-forge

**Comment by astrojuanlu** [comment]
xref https://github.com/kedro-org/kedro-viz/issues/2310
